hello-world/README.md

```diff
@@ -361,7 +361,7 @@ Now we are ready to define the `loop()` function and call it. Within the definit
 ensure that interval for function execution is set for 2 seconds:
```

Comment:
This should be updated as well (just for future reference, as the interval reduction is already merged to master.